geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegionClear.java

```diff
@@ -396,6 +399,17 @@ void doClear(RegionEventImpl regionEvent, boolean cacheWrite) {
       // Force all primary buckets to be created before clear.
       assignAllPrimaryBuckets();
 
+      Iterator<AsyncEventQueue> allAEQsIterator =
+          partitionedRegion.getCache().getAsyncEventQueues(false).iterator();
+      while (allAEQsIterator.hasNext()) {
```

Comment:
don't use an Iterator here. Instead do for (AsyncEventQueue asyncEventQueue: partitionedRegion.getCache().getAsyncEventQueues(false)) and then cast "asyncEventQueue" to an the Impl to call the internal method